content/api_en/environment/index.html

```diff
@@ -112,8 +112,6 @@ <h5 id="PDE">
       "sketch_YYMMDDa".</li>
     <li><em>Open...</em><br />
       Open a sketch in a new window.
-    <li><em>Sketchbook</em><br />
```

Comment:
Information about sketchbook button is also present after 'open recent' button. So, I removed this one.